### DISCUSSION TITLE
How to install Chromebrew (and get back sudo, if desired)

### DISCUSSION THREAD

**Discussion by CroutonIsFun** [dnschneid/crouton#5068] [General]
This is just one method. Installing Chromebrew requires that a command be typed correctly. But you can't sudo anymore in the crosh shell, and you can't copy and paste into VT-2, which is frecon.

Copy and paste the following line of text into the crosh shell, and press Enter. It will add an alias to your .bashrc file, enabling you to execute the command in VT-2:

echo 'alias ChromebrewInstall="bash <(curl -L git.io/vddgY) && . /.bashrc"'>>~/.bashrc

After running the above command, exit the shell with Ctrl-D, then re-enter the shell as before. This will update your environment variables. Now switch over to VT-2, log in as chronos, and start typing ChromebrewInstall. After typing only 2 or 3 letters, you can press Tab to autocomplete. When you press Enter, Chromebrew will be installed.

After the install is completed, you can log out of VT-2. Next (optional), go back to the crosh shell and enter this command:

crew install crew_sudo

After the above command, when you switch over to VT-2 and log in as chronos, a daemon will run, which will allow you to use sudo in the crosh shell.

**Comment by CroutonIsFun** [comment]
Will reopen later if appropriate...

**Comment by DennisLfromGA** [comment]
Which is it 🤔:

- #5065
or 
- #5068

I think you need to clarify the above or remove the the one that's invalid.

**Comment by CroutonIsFun** [comment]
> Which is it 🤔:
> 
> * [Do Not Install Chromebrew for Any Reason!!! #5065](https://github.com/dnschneid/crouton/discussions/5065)
>   or
> * [How to install Chromebrew (and get back sudo, if desired) #5068](https://github.com/dnschneid/crouton/discussions/5068)
> 
> I think you need to clarify the above or remove the the one that's invalid.

@DennisLfromGA 

I closed (attempted to close?) the one which says, do not install chromebrew. Is there any way to actually delete the post?

**Comment by DennisLfromGA** [comment]
No worries, I could delete it but instead chose to just lock it and mark it solved, hopefully this will let others know it's been resolved.
